node/src/main/scala/coop/rchain/node/api/WebApi.scala

```diff
@@ -107,9 +107,12 @@ object WebApi {
         .map(toDataAtNameResponse)
 
     def getDataAtPar(req: DataAtParRequest): F[RhoDataResponse] =
-      BlockAPI.getDataAtPar(req.blockHash, toPar(req), req.usePreStateHash).map {
-        case (pars, block) => RhoDataResponse(pars.flatMap(exprFromParProto), block)
-      }
+      BlockAPI
+        .getDataAtPar(toPar(req), req.blockHash, req.usePreStateHash)
+        .flatMap(_.liftToBlockApiErr)
+        .map {
+          case (pars, block) => RhoDataResponse(pars.flatMap(exprFromParProto), block)
```

Comment:
Please use `toExploratoryResponse` (renamed to `toRhoDataResponse`) because it has the same signature.

Reply:
Yes, thanks for your attentiveness! Done in [commit](https://github.com/rchain/rchain/pull/3584/commits/14d1f75f5a7c150b7a864319b7939e71d2c8d7fa).